rackcity/utils/log_utils.py

```diff
@@ -163,29 +167,19 @@ def log_power_action(user, power_action, related_asset):
     log.save()
 
 
-def log_network_action(user, action, asset_0, asset_1):
+def log_network_action(user, related_asset):
```

Comment:
there was no clear point in the creation/deletion of network connections where it made sense to call this log. so now network connection actions will be logged only via the asset they were done from

Reply:
do we need to have logging for bulk import? / will this support network bulk import?

Reply:
going to go back and perfect the bulk import logging and errors in one swoop later!